internal/services/ch/ch.go

```diff
@@ -22,6 +23,10 @@ type Client struct {
 
 const integPrefix = "dimo/integration/"
 
+// constantSignals are signals that never change. They tend to come from VIN decoding and should
+// not be taken as signs of an active connection.
+var constantSignals = []any{vss.FieldPowertrainTractionBatteryGrossCapacity}
```

Comment:
In telemetry API I am able to pass in a slice as a single arg to the the qm.WhereIn without the variadic expansion  
```suggestion
var constantSignals = []string{vss.FieldPowertrainTractionBatteryGrossCapacity}
```

Reply:
When I did string it freaked

Reply:
Oh you're proposing 
```
qm.WhereNotIn("name NOT IN ?", constantSignals)
```
Now I'm scared.